config/api.yml

```diff
@@ -35,6 +35,13 @@
     - :post
     - :patch
     - :delete
+    :gpppdo: &gpppdo
```

Comment:
@imtayadeway ok, so essentially every `OPTIONS` call should route to `base#options` unless otherwise specified (like I had prior to this)? Or does your refactor change this - will there still be a `base` controller?